addons/website_forum/models/forum.py

```diff
@@ -193,7 +195,7 @@ def _get_plain_content(self):
 
     content_link = fields.Char('URL', help="URL of Link Articles")
     tag_ids = fields.Many2many('forum.tag', 'forum_tag_rel', 'forum_id', 'forum_tag_id', string='Tags')
-    state = fields.Selection([('active', 'Active'), ('close', 'Close'), ('offensive', 'Offensive')], string='Status', default='active')
+    state = fields.Selection([('active', 'Active'), ('pending', 'Waiting Validation'), ('close', 'Close'), ('offensive', 'Offensive'), ('flagged', 'Flagged')], string='Status', default='active')
```

Comment:
Is it necessary to differentiate offensive from flagged ? In my mind
someone posts with enough karma: active
someone posts without enough karma: pending
then someone validate: active
someone flags: offensive / flagged (choose)
someone reset the flag: active or close the posts: close (with a reason).